config/routes.rb

```diff
@@ -9,11 +9,13 @@
   root 'users#index'
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show, :new, :create]
+    resources :posts, only: [:index, :show, :new, :create, :destroy]
   end
```

Comment:
- [ ] An error occurs while trying to create new posts as a result of your having some routes related to the posts which are repeating as shown in the image below. These repetitions occur on line 5 and 12. Please remove the former so that your application can work properly 👍 
![image](https://user-images.githubusercontent.com/46839548/190665684-4bfdfb28-6328-43ab-a8c3-ebca866abb43.png)